CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [2.13.0]
+
+## Added
+
+- Endpoint to set session grants based on session handle
+
+## Changed
+
+- Updated session endpoint to support grants
```

Comment:
Please see the format of the other items in the changelog and update this accordingly.